docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,266 @@
+file.
+
+### Import headers
+
+If a C library does not have a Clang module and instead consists of a set of headers, use the `headers` property to
+specify headers that should be imported:
+
+```none
+headers = curl/curl.h
+```
+
+#### Filter headers by globs
+
+You can filter headers by globs using filter properties from the `.def` file. To include declarations from headers,
+use the `headerFilter` property. If a header matches any of the globs, its declarations are included in the bindings.
+
+The globs are applied to the header paths relative to the appropriate include path elements,
+for example, `time.h` or `curl/curl.h`. So if the library is usually included with `#include <SomeLibrary/Header.h>`,
+it would probably be correct to filter headers with the following filter:
+
+```none
+headerFilter = SomeLibrary/**
+```
+
+If `headerFilter` is not provided, all the headers are included. However, we encourage you to use `headerFilter`
+and specify the glob as precisely as possible. In this case, the generated library contains only the necessary
+declarations. It can help avoid various issues when upgrading Kotlin or tools in your development environment.
+
+#### Exclude headers
+
+To exclude specific headers, use the `excludeFilter` property. It can be helpful to remove redundant or problematic
+headers and optimize compilation, as declarations from the specified headers are not included into the bindings:
+
+```none
+excludeFilter = SomeLibrary/time.h
+```
+
+> If the same header is both included with `headerFilter`, and excluded with `excludeFilter`, the specified header will
+> not be included into the bindings.
+>
+{type="note"}
+
+### Import modules
+
+If an Objective-C library has a Clang module, use the `modules` property to specify the module that should be imported:
+
+```none
+modules = UIKit
+```
+
+### Pass compiler and linker options
+
+Use the `compilerOpts` property to pass options to the C compiler, which is used to analyze headers under the hood.
+To pass options to the linker, which is used to link final executables, use `linkerOpts`. For example:
+
+```none
+compilerOpts = -DFOO=bar
+linkerOpts = -lpng
+```
+
+Target-specific options only applicable to the certain target can be specified as well:
+
+```none
+compilerOpts = -DBAR=bar
+compilerOpts.linux_x64 = -DFOO=foo1
+compilerOpts.macos_x64 = -DFOO=foo2
+```
+
+With such a configuration, headers will be analyzed with `-DBAR=bar -DFOO=foo1` on Linux and with `-DBAR=bar -DFOO=foo2`
+on macOS. Note that any definition file option can have both common and platform-specific parts.
+
+### Ignore specific functions
+
+Use the `excludedFunctions` property to specify a list of the function names that should be ignored. This may be
+helpful in case a function declared in the header is not generally guaranteed to be really callable, and
+it's often hard or impossible to figure this out automatically. You can also use this property to work around a bug
```

Comment:
```suggestion
Use the `excludedFunctions` property to specify a list of the function names that should be ignored. This can be
useful if a function declared in the header isn't guaranteed to be callable, and
it's difficult or impossible to determine this automatically. You can also use this property to work around a bug
```